app/facades/foods_facade.rb

```diff
@@ -0,0 +1,12 @@
+class FoodsFacade
+  def self.total_hits(params)
+    results = FoodsService.get_search(params)
+    results[:totalHits]
+  end
+
+  def self.food_results(params)
+    FoodsService.get_search(params)[:foods][0..9].map { |data| Food.new(data) }
+    # results = FoodsService.get_search(params)
+    # wip = results[:foods].find { |result| result[:ingredients].include?(params) }.limit(10)
```

Comment:
Great start for filtering out empty ingredients! Apologies for the confusion -- this wasn't actually necessary and also would've been an extension, but you've got the right idea with this filter functionality.